docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,225 @@
+        {{else}}
+            <div class="header-logo-image-container">
+                <img class="header-logo-image" src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
+            </div>
+        {{/if}}
+    {{else}}
+        <span class="header-logo-text">{{settings.store_logo.title}}</span>
+    {{/if}}
+</a>
+```
+Add or update the image alt text to provide descriptive text for your store logo.
+
+```html
+alt = "BigCommerce store logo" 
+```
+
+NOTE: You can update the `<title>` tag with the same text as the `<alt>` tag to create a tooltip, or remove it if you do not need it.
+
+**Page Builder example**
+
+You can add alt text to the `<img>` element in the Image Widget of Page Builder or the `cornerstone/assets/img/` directory.
+
+Follow the steps below to add alt text to an image using Page Builder:
+1. Go to **Storefront** > **My theme** in the control panel, then click **Customize**.
+2. Launch Page Builder and select the image widget.
+3. Enter the alt text.
+4. Click **Save**.
+
+![Image Widget](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility-image-widget.png "Image Widget")
+
+### Aria-label attribute
+There are other options for providing text for images. Developers can use the aria-label attribute, a [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) technique that provides assistive technology with the aria-label text instead of the image alt text. 
+
+The aria-label attribute uses an ID reference value that matches the ID element attribute to associate an element with text. Screen readers use the text of one or more identified elements that can be referenced elsewhere on the page.
+
+**Popup modal text example**
+
+In Cornerstone, put the following code at the bottom of your stencil template html:
+
+```html
+<div id="elementID" class="modal modal--large" data-reveal>
+    <a href="#" class="modal-close" aria-label="Popup text" role="button">
+        <span aria-hidden="true">&#215;</span>
+    </a>
+    <div class="modal-content"></div>
+    <div class="loadingOverlay"></div>
+</div>
+```
+Add a button on your page and use the following code:
+```html
+<!--open custom modal-->
+       <div class="modal-button-container">
+                <a class="button" href="#elementID" data-reveal-id="elementID">Descriptive text</a>
+        </div>
+```
+
+## Text accessibility
+Text accessibility involves making content readable and understandable. You can achieve text accessibility using the following methods:
+* [Color contrast](#color-contrast)
+* [Headings](#headings)
+* [Font size and text alignment](#font-size-and-text-alignment)
+* [Text links](#text-links)
+
+### Color contrast
+Color contrast is an essential factor in visual accessibility.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
```

Comment:
Consider adding the contents of this note to the previous sentence. It's an important callout that we don't want people to skip over.